client/src/components/Feed/Post.js

```diff
@@ -156,4 +191,10 @@ const Post = ({ post }) => {
   );
 };
 
-export default Post;
+const mapStateToProps = ({ session }) => {
+  return {
+    isAuthenticated: session.isAuthenticated,
```

Comment:
Can just be `isAuthenticated ` through deconstructing

Reply:
Done.